src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/navigate-modal/NavigateLocationModal.tsx

```diff
@@ -0,0 +1,315 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FormEvent, useRef, useState } from 'react';
+import classNames from 'classnames';
+
+import { useAppSelector, useAppDispatch } from 'src/store';
+import { useGenomeKaryotypeQuery } from 'src/shared/state/genome/genomeApiSlice';
+import useGenomeBrowser from 'src/content/app/genome-browser/hooks/useGenomeBrowser';
+
+import { getBrowserActiveGenomeId } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSelectors';
+
+import { getNumberWithoutCommas } from 'src/shared/helpers/formatters/numberFormatter';
+import {
+  validateRegion,
+  type RegionValidationErrors
+} from 'src/content/app/genome-browser/helpers/browserHelper';
+
+import Input from 'src/shared/components/input/Input';
+import { PrimaryButton } from 'src/shared/components/button/Button';
+import Tooltip from 'src/shared/components/tooltip/Tooltip';
+import Select from 'src/shared/components/select/Select';
```

Comment:
We are using a different select component now. The one you are importing looks like this:

![image](https://user-images.githubusercontent.com/6834224/208409434-93e325a2-5585-43cf-8327-30317fb5a88b.png)

whereas what the XD shows is this:

![image](https://user-images.githubusercontent.com/6834224/208409530-cebba38e-3100-4e69-a8b1-e1a033aad9de.png)

[Here](https://ensembl.github.io/ensembl-client/storybook/index.html?path=/story/components-shared-components-simpleselect--simple-select-default-story) is what you are after.

Reply:
✅

Reply:
Do we start with 1? Or should we start with an unselectable empty option "Select"?


https://user-images.githubusercontent.com/6834224/208935285-9b4439af-5939-4649-bbaf-b9a9707c7577.mp4

Reply:
That's done now ✅ 

The option selected is not reset for now as that seems to be causing other issues at the moment. Will look into this later.

Reply:
![image](https://user-images.githubusercontent.com/6834224/209304098-c3124de6-991a-44dc-90fd-11528f805386.png)

I am seeing blank space now.

Reply:
Argh I forgot to add the text `Select` as the label.

Reply:
✅